test/built-ins/Temporal/Instant/prototype/add/throw-range-error-from-AddInstant.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
```

Comment:
Consider combining with https://github.com/tc39/proposal-temporal/blob/main/polyfill/test/Instant/prototype/add/result-out-of-range.js